cli/cli/cmds_volume.go

```diff
@@ -339,7 +344,11 @@ func (c *CLI) initVolumeCmds() {
 			// volumes that are attached or can be attached should be valid
 			// candidates for a mount operation
 			result, err := c.lsVolumes(
-				args, volumeStatusAttached, volumeStatusAvailable)
+				args,
+				requestAttachments(
+					apitypes.VolumeAttachmentsMine,
+					apitypes.VolumeAttachmentsAttached,
+					apitypes.VolumeAttachmentsUnattached))
```

Comment:
sorry to create a new review, but I hit a problem with testing today. I'm not sure if the solution is a change to rex, or to libstorage, but as it stands with LS 0.3.3-rc3, you can't request both attached and unattached volumes at the same time.

this is because if you specific attached volumes, any volumes that are unattached are filtered out: https://github.com/codedellemc/libstorage/blob/master/api/server/router/volume/volume_routes.go#L181-L187

The inverse turns out to be true, but the check for attached happens before checking for unattached.